### PR TITLE
Validate minLength & maxLength for some body params

### PR DESCRIPTION
**Pull request by shrikrishnaholla**
Restricts the length of the name of the orgs, name of the projects, name of teams and name of environments.
min length: 3, max length: 15

### CODE REVIEW

File: api/envs/create.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ module.exports = function (app, db) {
     var errs = app.utils.need(req, ['name', 'description']);
     var name = req.body.name;
 
-    if (typeof name != 'string' || name.match(/[a-z0-9][a-z0-9\ ]*[a-z0-9]/i)[0] != name) {
+    if (typeof name != 'string' || name.length < 3 || name.length > 15 ||
+     name.match(/[a-z0-9][a-z0-9\ ]*[a-z0-9]/i)[0] != name) {
```
Comment (pksunkara) [inline api/envs/create.js]: No need for 2 lines here. Just 1 line is enough. Similarly the other files below.

**Comment by pksunkara** [comment]
Could you remove the `[#200](...)` from the commit message? It is unnecessary and commit message length becomes too big.

**Comment by pksunkara** [comment]
Other than that, this is great work. Add length validation for User's username and I will be able to merge this. :smile:

**Comment by shrikrishnaholla** [comment]
Great! No problem! :+1:

File: api/envs/create.js (hunk 1 above)
Comment (shrikrishnaholla) [inline api/envs/create.js]: Doesn't it get too lengthy? 113 columns!

Comment (pksunkara) [inline api/envs/create.js]: No issue. We don't follow that convention.

Comment (shrikrishnaholla) [inline api/envs/create.js]: Okay then

**Comment by pksunkara** [comment]
Merged.

**Pull request closed without merging**